src/video_core/vulkan_common/vulkan_memory_allocator.cpp

```diff
@@ -311,6 +330,20 @@ std::optional<u32> MemoryAllocator::FindType(VkMemoryPropertyFlags flags, u32 ty
     return std::nullopt;
 }
 
+void MemoryAllocator::TickFrame() {
+    const auto now{std::chrono::steady_clock::now()};
+    if (now - gc_timer >= ALLOCATION_TICK) {
+        for (s64 x = allocations.size() - 1; x > 0; --x) {
```

Comment:
```suggestion
        for (s64 x = static_cast<s64>(allocations.size()) - 1; x > 0; --x) {
```

Otherwise this is performing an implicit sign conversion